lib/Service/VaultService.php

```diff
@@ -41,7 +41,7 @@ public function __construct(VaultMapper $vaultMapper) {
 	/**
 	 * Get vaults from a user.
 	 * @param $userId
-	 * @return \OCA\Passman\Db\Vault[]
+	 * @return Entity[]
```

Comment:
No longer a vault array type?

Reply:
no, we talked about it, without the "IDE improvements" its all Entity and Entity[]